app/views/reports/regions/details.html.erb

```diff
@@ -87,16 +90,16 @@
                   <%= link_to resource.send(:full_name), send(:admin_user_path, resource, period: @period) %>
                 </td>
                 <td class="ta-center">
-                  <%= number_with_delimiter(dash_if_zero(@dashboard_analytics.dig(resource.id, :total_registered_patients))) %>
+                  <%= number_or_dash_with_delimiter(@dashboard_analytics.dig(resource.id, :total_registered_patients)) %>
```

Comment:
Technically, it's `number_with_delimiter_or_dash`. While we're consolidating this, shall we name the method on what it does, not how it does it? Eg. `format_indicator`?

Reply:
I wanted to keep this consistent with the Rails view helper `number_with_delimiter` because our little helper is essentially just a wrapper around it.